Sources/Commands/SwiftPackageTool.swift

```diff
@@ -58,15 +73,15 @@ public struct SwiftPackageTool: ParsableCommand {
             ComputeChecksum.self,
             ArchiveSource.self,
             CompletionTool.self,
-        ],
-        helpNames: [.short, .long, .customLong("help", withSingleDash: true)])
-
-    @OptionGroup()
-    var swiftOptions: SwiftToolOptions
-
-    public init() {}
-
-    public static var _errorLabel: String { "error" }
+        ]
+        
+        if InitPackage.createPackageMode == .new {
+            subCommands.insert(Create.self, at: 6)
```

Comment:
Would it make sense to derive these indices from looking up `Init.self` in the array?  If not, it would be good to have a comment up in that array, since it's very easy otherwise to add a subcommand without realizing that this hardcoded index has to be updated.